Dockerfile

```diff
@@ -4,27 +4,18 @@ MAINTAINER Code Climate <hello@codeclimate.com>
 RUN apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg && \
     echo "deb http://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
     apt-get update && \
-    apt-get install -y yarn
+    apt-get install -y git jq yarn
 
 WORKDIR /usr/src/app
-COPY package.json yarn.lock /usr/src/app/
+COPY bin/docs ./bin/docs
+COPY package.json yarn.lock ./
 
-RUN yarn install
+RUN yarn install && \
+    bin/docs && \
+    apt-get purge --yes git jq yarn && \
```

Comment:
Purging these packages in a separate layer from where they're installed is sort of useless: they're still in the earlier layers, so they still need to pulled.

If we can tolerate the size of these packages, we should just skip the `purge`. If they have a significant impact on overall size, then in order to really keep size down we'd need to do the `apt-get install` in the same layer as the other commands here, which seems contrary to your goal.